Dutch.json

```diff
@@ -0,0 +1,58 @@
+{
+  "version": 3,
+  "DeathTextString": "werd gedood door",
+  "SCP008String": "SCP-008",
+  "UnknownString": "een onbekende oorsprong",
+  "PlayerString": "een andere speler",
+  "SCP049String": "SCP-049",
+  "SCP0492String": "SCP-049-2",
+  "SCP096String": "SCP-096",
+  "SCP106String": "SCP-106",
+  "SCP173String": "SCP-173",
+  "SCP914String": "SCP-914",
+  "SCP939String": "SCP-939",
+  "SCP966String": "SCP-966",
+  "SCP012String": "SCP-012",
+  "TeslaGateString": "een tesla poort",
+  "GASString": "giftig gas",
+  "SuicideString": "zichzelf",
+  "FallDamageString": "zwaartekracht",
+  "DisconnectedString": "ophouden te bestaan",
+  "NoBlinkOnString": "Je oogleden zijn gewist",
+  "NoBlinkOffString": "Je oogleden zijn terug gekeerd",
+  "ReloadGroupConfirmedString": "Servergroepen zijn opnieuw geladen.",
+  "FlyOnString": "Vlieg nu weg! Vlieg nu weg! Vlieg nu weeeeeeg!",
+  "FlyOffString": "Je zou nooit een betere tijd vinden! Om je standpunt in te nemen, komt alles goed!",

```

Comment:
Needs to be "Je zult nooit een betere tijd vinden! Om je standpunt te verdedigen, komt het goed!"